src/routes/about/+page.svelte

```diff
@@ -24,11 +24,10 @@
 			<a href="https://twitter.com/feltcoop">Twitter</a>, and
 			<a href="https://fosstodon.org/@feltcoop">Mastodon</a>) is an operator of
 			<a href="https://github.com/feltjs/felt-server">felt-server</a>
-			instances, providing hosting services for end-users. The goal is to create a collectively-owned
-			platform, free of ads and investors, to support communities that are human-scale, self-governed,
-			consentful, private by default, and deeply customizable and programmable. We provide free and open
-			source tools to design our social spaces and share our creations with others, or that's the plan...
-			check back here in Q1 2023, and for updates see
+			instances, providing hosting services for end-users. Our mission is to make our lives better with
+			technology. The goal is to create a collectively-owned platform, free of ads and investors, to
```

Comment:
```suggestion
			technology. As part of that mission we plan to create a collectively-owned platform, free of ads and investors, to
```
Just a thought since the "goal" sentence seems a little disconnected with the new mission phrasing.

Reply:
good stuff